documentation/config_docs.md

```diff
@@ -238,3 +242,25 @@ You can optionally provide a **status condition** to specify additional requirem
     }
 }
 ```
+
+## Project Owners
+
+In GitHub, ["code owners"](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) are the users/teams whose review is automatically requested when a PR modifying code in a given directory is opened. **Project owners** behave similarly, with two differences.
+
+- Owners are defined _per PR label_, instead of _per directory_ (or directory pattern)
```

Comment:
future improvement : s/instead/in addition to/

Reply:
then for flexibility maybe better change format to

```
"project_owners": [
  {
    "label": "Label 1",
    "owners": ["user1", "user2", "org/team1"]
  }, ...
]
```

Reply:
so verbose but probably yes